pyflow/packages/RCDT/Core/node_loader.py

```diff
@@ -27,19 +27,21 @@ def get_pyflow_nodes_from_ros_services(
     service_definitions: list[ServiceDefinition],
 ) -> dict:
     return {
-        service_definition.service_type.__name__: create_class_from_service(
-            service_definition
-        )
+        service_definition.pyflow_name: create_class_from_service(service_definition)
         for service_definition in service_definitions
```

Comment:
Geen nieuwe code, maar ik ben in de regel niet zo'n fan van dit soort postfixing syntax. Misschien de moeite om een keer een discussie over te hebben.